initialise/MirahezeFunctions.php

```diff
@@ -262,15 +279,15 @@ public static function isMissing() {
 		return !in_array( self::getCurrentDatabase(), $wgConf->wikis );
 	}
 
-	public function getCacheArray() {
+	public function getCacheArray(): array {
 		// If we don't have a cache file, let us exit here
 		if ( !file_exists( self::CACHE_DIRECTORY . '/' . $this->dbname . '.json' ) ) {
 			return false;
```

Comment:
This doesn't seem to match the defined return type(`array`).

Reply:
Oh, thanks!